operator/pkg/utils/imageprovider/imageprovider.go

```diff
@@ -19,6 +19,7 @@ var (
 		"1.19": kubeVersion119Tag,
 		"1.20": kubeVersion120Tag,
 		"1.21": kubeVersion121Tag,
+		"1.22": kubeVersion122Tag,
```

Comment:
Did we confirm a 1.22 cluster is launched successfully? In EKS we enabled CCM in 1.22 and started the migration in 1.21. Do we plan to enable CCM in KIT?